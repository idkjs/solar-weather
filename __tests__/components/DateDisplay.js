import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import DateDisplay from '../../lib/js/app/components/dateDisplay';

const date = new Date(1515069527 * 1000);

test('renders correctly', () => {
  const tree = renderer.create(<DateDisplay
    condition="sunny"
    timestamp={date}
    timezone="America/New_York"
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
