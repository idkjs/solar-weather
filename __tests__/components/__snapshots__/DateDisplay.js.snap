// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityLiveRegion={undefined}
  accessibilityTraits={undefined}
  accessibilityViewIsModal={undefined}
  accessible={undefined}
  collapsable={undefined}
  hitSlop={undefined}
  importantForAccessibility={undefined}
  needsOffscreenAlphaCompositing={undefined}
  onAccessibilityTap={undefined}
  onLayout={undefined}
  onMagicTap={undefined}
  pointerEvents={undefined}
  removeClippedSubviews={undefined}
  renderToHardwareTextureAndroid={undefined}
  shouldRasterizeIOS={undefined}
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "transparent",
      "height": 50,
      "left": "5.00%",
      "position": "absolute",
      "top": "2.50%",
      "width": 200,
    }
  }
  testID={undefined}
>
  <Text
    accessible={true}
    adjustsFontSizeToFit={undefined}
    allowFontScaling={true}
    ellipsizeMode="tail"
    minimumFontScale={undefined}
    numberOfLines={undefined}
    onLayout={undefined}
    onLongPress={undefined}
    onPress={undefined}
    pressRetentionOffset={undefined}
    selectable={undefined}
    selectionColor={undefined}
    style={
      Object {
        "color": "#FFF",
        "lineHeight": 20,
      }
    }
    suppressHighlighting={undefined}
    testID={undefined}
    textBreakStrategy={undefined}
  >
    7:38am
  </Text>
  <Text
    accessible={true}
    adjustsFontSizeToFit={undefined}
    allowFontScaling={true}
    ellipsizeMode="tail"
    minimumFontScale={undefined}
    numberOfLines={undefined}
    onLayout={undefined}
    onLongPress={undefined}
    onPress={undefined}
    pressRetentionOffset={undefined}
    selectable={undefined}
    selectionColor={undefined}
    style={
      Object {
        "color": "#FFF",
        "lineHeight": 20,
      }
    }
    suppressHighlighting={undefined}
    testID={undefined}
    textBreakStrategy={undefined}
  >
    Jan 04
  </Text>
</View>
`;
